### DISCUSSION TITLE
Book covers?

### DISCUSSION THREAD

**Discussion by mmol67** [aedocw/epub2tts#190] [Ideas]
Hello.

Maybe you can add a flag to add a book covers (or art) to the audio file.

`--covers cover.jpg,backcover.jpg`

being the first one the main cover.

Regards,

**Comment by aedocw** [comment]
I'd be happy to add this, but I'm not sure how to embed the cover art into an m4b. If you've got good pointers, please share. Otherwise I'll eventually figure it out :)

**Comment by aedocw** [comment]
Looks like this is pretty easy actually, I'll put this into a testing branch in a little bit. Basically just have to do:

```
from mutagen import mp4
m4b = mp4.MP4(m4b_filename)
cover_image = open(cover_image, "rb").read()
m4b["covr"] = [mp4.MP4Cover(cover_image)]
m4b.save()
```

**Comment by aedocw** [comment]
Really great idea, I had thought of this a long time ago but never felt the urge to actually do anything about it. It has now been merged with https://github.com/aedocw/epub2tts/pull/192
